nntrainer/src/neuralnet.cpp

```diff
@@ -448,12 +448,16 @@ int NeuralNetwork::train_run() {
 
     int count = 0;
 
+    TensorDim inputDim = getInputDimension();
+    TensorDim labelDim{batch_size, 1, 1,
```

Comment:
why use just width of layers.back() than its full output dimension?

Reply:
Right. I'll change